src/view/components/checkboxList.jsx

```diff
@@ -38,20 +38,16 @@ class CheckboxList extends React.Component {
   render() {
     const { options, value, className } = this.props;
     const listItems = options.map(option => {
-      let optionValue;
-      let optionLabel;
-
-      if (typeof option === "string") {
```

Comment:
This was so we could just pass an array of values for the `options` props like:

`options={["cookie", "cake", "ice cream"]}`

We're not using this and I don't know that it buys us much, so I took it out for simplicity.